app/Http/Controllers/WorkflowController.php

```diff
@@ -0,0 +1,179 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\EvidencioAPI;
+use App\Workflow;
+use App\LoadedEvidencioModel;
+use App\Step;
+use App\Field;
+use App\Option;
+use Illuminate\Support\Facades\Gate;
+use Illuminate\Validation\UnauthorizedException;
+
+/**
+ * WorkflowController class, handles database- and API-calls for Workflowpage.
+ */
+class WorkflowController extends Controller
+{
+
+    /**
+     * Show the application dashboard.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index($workflowId)
+    {
+        $result = WorkflowController::loadWorkflow($workflowId);
+        return view('workflow')->with('result', $result);
+    }
+
+    /**
+     * Loads a workflow from the database based on the workflowId
+     *
+     * @param Number $workflowId
+     * @return Array
+     */
+    public function loadWorkflow($workflowId)
+    {
+        $retObj = [];
+        $usedVariables = [];
+        $workflow = Workflow::find($workflowId);
```

Comment:
If you type hinted this then Laravel can do this automatically for you

Reply:
Could you please describe in more detail what you mean?

Reply:
<img width="647" alt="screen shot 2018-06-27 at 21 20 10" src="https://user-images.githubusercontent.com/6588838/41994793-ec60942e-7a4f-11e8-82ec-4c9f0c91b367.png">

Reply:
So if you type-hint your function like so: `function (Workflow $workflow)`, there is no need to do a `Workflow::find($id)` as Laravel automatically searches for an entity using the id.